### DISCUSSION TITLE
Show `draft_docs` pages in navigation while serving

### DISCUSSION THREAD

**Discussion by joapuiib** [mkdocs/mkdocs#3827] [Q&A]
Hello.
I'd like to know if it is possible to show draft pages (configured via [`draft_docs`](https://www.mkdocs.org/user-guide/configuration/#draft_docs)) in the navigation while serving the site.

Thanks.

**Comment by smainz** [comment]
This would be very convenient for authors.Otherwise they have to copy the link displayed.
